contrib/scrap-ec2-sizes.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python
+#
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+"""
+This script downloads and parses AWS EC2 from https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/current/index.json.
+It writes a Python module with constants about EC2's sizes and regions.
+
+Use it as following:
+    $ python scrap-ec2-sizes.py > ../libcloud/compute/constants.py
```

Comment:
It would be better to document this as `python contrib/scrap-ec2-sizes.py > libcloud/compute/constants.py`.

We also need to update "Making a release" in docs/committer_guide.rst. I suggest adding a "0. Update committed files" section with the CHANGES and libcloud/__init__.py lines from the "1. Pre-release check list" section.

Reply:
Sorry, I'm not sure to understand, could you rephrase it ?

Reply:
Something like this:

```
diff --git a/docs/committer_guide.rst b/docs/committer_guide.rst
index 0f998088..4128e962 100644
--- a/docs/committer_guide.rst
+++ b/docs/committer_guide.rst
@@ -91,12 +91,17 @@ Making a release (for release managers)
 This section contains information a release manager should follow when
 preparing a release.

-1. Pre-release check list
+0. Update committed files
 ~~~~~~~~~~~~~~~~~~~~~~~~~

-* Make sure tests pass on all the supported Python versions (``tox``)
 * Make sure ``CHANGES`` file is up to date
 * Make sure ``__version__`` string in ``libcloud/__init__.py`` is up to date
+* Update constants. ``python contrib/scrap-ec2-sizes.py > libcloud/compute/constants.py``
+
+1. Pre-release check list
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Make sure tests pass on all the supported Python versions (``tox``)
 * Remove the ``tox`` directory with ``rm -rf .tox``
 * Remove the _secrets_ file with ``rm libcloud/test/secrets.py``
 * Remove leftover builds from previous releases. ``rm -f dist/apache*``
```